docker/keto/relation-tuples/authorities.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "namespace":"members",
```

Comment:
What does this mean by "members"? Is it keto domain or ?

Reply:
It is a kind of scope for ACLs in Keto. Keto organizes relation tuples based on the `namespace`. But for the sake of this PR, a separate JSON file for relation tuples is not needed. It is deleted for now.